samaritan/src/components/identificationSettings/identificationSettings.tsx

```diff
@@ -1,8 +1,94 @@
 import React from "react";
+import { Input } from "antd";
+import { Typography } from "antd";
+import { Upload, message, Button, Icon } from "antd";
+import { Row, Col, Avatar, Layout, Menu } from "antd";
+import { Form } from "antd";
+import "./identificationSettings.css";
+
+const { Title } = Typography;
+const { Dragger } = Upload;
+const { Content, Sider, Header } = Layout;
+
+const props = {
+  name: "file",
+  multiple: true,
+  action: "https://www.mocky.io/v2/5cc8019d300000980a055e76",
+  onChange(info) {
+    const { status } = info.file;
+    if (status !== "uploading") {
+      console.log(info.file, info.fileList);
+    }
+    if (status === "done") {
+      message.success(`${info.file.name} file uploaded successfully.`);
+    } else if (status === "error") {
+      message.error(`${info.file.name} file upload failed.`);
+    }
+  }
+};
 
 class IdentificationSettings extends React.Component {
+  state = {
+    collapsed: false
+  };
+  onCollapse = () => {
+    this.setState({
+      collapsed: !this.state.collapsed
+    });
+  };
   render() {
-    return <>IdentificationSettings</>;
+    return (
+      <>
+        <Layout style={{ minHeight: "100vh" }}>
```

Comment:
don't need the dashboard code on this, this component is gonna lay inside some bigger settings component so just need the form information (line 71-89), so get rid of the dashboard code